version: "3"

services:
  app:
    # Not working yet with TLS; probably a bug in the distributed image
    image: portainer/portainer:1.13.3
    ports:
    - 9000:9000
    volumes:
    - ${SHARE_PATH}/portainer/data:/data
    deploy:
      placement:
        constraints:
        - node.labels.swarm-sync-member == true
    networks:
      app_net:

networks:
  app_net:
    external: true
